src/test/java/instance/adminViewIT.java

```diff
@@ -0,0 +1,159 @@
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+@FixMethodOrder(MethodSorters.JVM)
+public class adminViewIT {
+    
+    private static WebDriver driver;
+    private static String baseUrl = "https://localhost:9443/instance/";
+    private static JavascriptExecutor js;
+
+    @BeforeClass
+    public static void setupClass() throws IOException {
+        // Manages the browser driver binary for us
+        WebDriverManager.chromedriver().setup();
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--ignore-certificate-errors", "--no-sandbox", "--disable-dev-shm-usage", "--disable-gpu");
+        options.setHeadless(true);
+        driver = new ChromeDriver(options);
+        driver.get(baseUrl);
+
+        js = (JavascriptExecutor) driver;
+
+        String multiInstanceJSON = new String(Files.readAllBytes(Paths.get("src", "test", "resources", "multipleInstances.json")), StandardCharsets.UTF_8);
+
+        // execute javascript functions to set mock data
+        js.executeScript("setInstanceSelections(JSON.parse(arguments[0]));", multiInstanceJSON);
+    }
+
+    @AfterClass
+    public static void teardown() {
+        if (driver != null) {
+            driver.quit();
+        }
+    }
+
+    @Test
+    public void isNotAdminIt(){
+        js.executeScript("fetchInstanceAdmins({'isAdmin':false});");
+        WebElement hiddenAdminList = driver.findElement(By.id("instance-accordion-admin-view"));
+        System.out.print("ssk" + hiddenAdminList.getText());
+
+        assertEquals("hides accordion admin list from non-admin", false, hiddenAdminList.isDisplayed());
+    }
+
+    @Test
+    public void isAdminIt() throws IOException {
+        String adminMembersJSON = new String(Files.readAllBytes(Paths.get("src", "test", "resources", "/adminJsons/adminMembers.json")), StandardCharsets.UTF_8);
+        js.executeScript("fetchInstanceAdmins({isAdmin:true});");
+        js.executeScript("updateInstanceAdminView(JSON.parse(arguments[0]));", adminMembersJSON);
+
+        WebElement accordionAdminList = driver.findElement(By.id("instance-accordion-admin-view"));
+
+        assertEquals("shows accordion admin list of admins", true, accordionAdminList.isDisplayed());
+    }
+
+    @Test
+    public void hasCorrectNumAdminIT() throws IOException {
+        List<WebElement> adminList = driver.findElements(By.className("instance-admin-names"));
+        int expectedNumAdmins = 3;
+        int actualNumAdmins = adminList.size();
+
+        assertEquals("has correct Number of admins", expectedNumAdmins, actualNumAdmins);
+    }
+
+    @Test
+    public void hasCorrectAdminListIT() throws IOException {
+        List<WebElement> adminList = driver.findElements(By.className("instance-admin-names"));
```

Comment:
for this test I wouldn't worry about _order_ we didn't write any code to make sure things appeared in a specific order. I would change this to just check that all 3 exist (in any order) so maybe compare 2 arrays for the equalness, or if you find a better way go for it